mod.test.ts

```diff
@@ -176,6 +176,99 @@ Deno.test("sleep command", async () => {
   assertEquals(end - start > 190, true);
 });
 
+Deno.test("test command", async (t) => {
+  await Deno.writeFile('zero.dat', new Uint8Array());
+  await Deno.writeFile('non-zero.dat', new Uint8Array([242]));
+  if (Deno.build.os !== 'windows') {
```

Comment:
Do you have developer mode enabled on your machine? https://blogs.windows.com/windowsdeveloper/2016/12/02/symlinks-windows-10/ (I use Windows 10 btw)

That said, I think excluding this test on Windows is fine.

Reply:
Looks like its failing in GH actions anyway.  I don't technically need the symlink stuff for my current use case, but, it seems like this should work?

Reply:
I feel like it should work too. I'm just looking into it.

Reply:
Oh, Deno.stat always follows symlinks. I'm just looking into an alternative.